### PR TITLE
feat: add MRU, SLE, STN currency support

### LINKED ISSUE #142: Lack of MRU, SLE and STN currency support

### PR DESCRIPTION
**Pull request by batuhansk**
Resolves #142

### CODE REVIEW

**Comment by Rhymond** [comment]
Thanks @batuhansk, could you leave a reference where you got details about these added currencies?

**Comment by batuhansk** [comment]
Sure, these are the references.

* https://www.iban.com/currency-codes
* https://en.wikipedia.org/wiki/ISO_4217
* https://en.wikipedia.org/wiki/Currency_symbol

File: currency.go [hunk 1]
```diff
@@ -179,11 +180,13 @@ var currencies = Currencies{
 	SGD: {Decimal: ".", Thousand: ",", Code: SGD, Fraction: 2, NumericCode: "702", Grapheme: "$", Template: "$1"},
 	SHP: {Decimal: ".", Thousand: ",", Code: SHP, Fraction: 2, NumericCode: "654", Grapheme: "\u00a3", Template: "$1"},
 	SKK: {Decimal: ".", Thousand: ",", Code: SKK, Fraction: 2, NumericCode: "", Grapheme: "Sk", Template: "$1"},
+	SLE: {Decimal: ".", Thousand: ",", Code: SLE, Fraction: 2, NumericCode: "694", Grapheme: "Le", Template: "1 $"},
```
Comment (Rhymond) [inline currency.go:183]: NumericCode is `925`

CleanShot 2024-04-02 at 10 20 38@2x

Comment (batuhansk) [reply currency.go:183]: Just updated it. It seems that the iban.com still shows the SLE (old Leone)

**Review by Rhymond — APPROVED** [review]

**Pull request merged**